src/dex-ui-components/base/Inputs/FileUploader.tsx

```diff
@@ -0,0 +1,75 @@
+import { useCallback } from "react";
+import { useDropzone } from "react-dropzone";
+import { Card, Text } from "@chakra-ui/react";
+import { Color } from "../../../dex-ui-components";
+interface File extends Blob {
+  readonly lastModified: number;
+  readonly name: string;
+  readonly webkitRelativePath: string;
+}
+
+const defaultFileUploaderStyles = {
```

Comment:
To stick with the ChakraUI styling config pattern, you can add these styles as a `variant` in the `CardStyles` object in **dex-ui-components/base/Card/styles.ts**.

`
export const CardStyles: ComponentStyleConfig = {
...
variants: {
"file-uploader": {
...styles
}
}
...
}
`

`
<Card {...getRootProps()} variant="file-uploader" style={{ ...fileUploaderStyles }}>
`